src/main/scala/extractionStreams/WikiExtractionStream.scala

```diff
@@ -0,0 +1,14 @@
+package extractionStreams
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.RunnableGraph
+
+/**
+ * Created by nico on 13/07/15.
+ */
+abstract class WikiExtractionStream[T] {
+  val stream: RunnableGraph[T]
+  def run()(implicit materializer: Materializer): Unit = {
```

Comment:
implicit means the materializer is available in all following function calls?

maybe you have a short tutorial on reactive streams for us?